source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -584,8 +622,8 @@ current monitoring connection. (See `Drivers cancel in-progress monitor checks`_
 Polling Protocol
 ''''''''''''''''
 
-The polling protocol is used to monitor MongoDB <= 4.4 servers. The client
-`checks`_ a server with a hello or legacy hello command and then sleeps for
+The polling protocol is used to monitor MongoDB <= 4.4 servers or when `streaming is disabled`_.
```

Comment:
Should this be "<= 4.4" or "< 4.4" ? It looks like streaming could be enabled for 4.4 versions of the server, testing on 4.4.24 it looks like the hello command contains the topology version.

Reply:
Good catch. Fixed.